frontend/src/employee-frontend/components/messages/types.ts

```diff
@@ -74,13 +74,13 @@ export const isGroupMessageAccount = (
 
 export interface Message {
   id: UUID
-  senderId: UUID
   senderName: string
   sentAt: Date
   readAt: Date | null
   title: string
   content: string
   receivers: UUID[]
+  type: MessageType
 }
```

Comment:
This interface should reflect the data we get from the backend (`thread.messages`). Type and title are properties of thread. It seems like `Message` and `MessageBody` types have melded. We should be using `MessageBody` or a derivative of it when creating a new message.

Reply:
Message should only include properties that are found in the backend. https://github.com/espoon-voltti/evaka/blob/52a8f0faca9d6dcca4c2201c389daa86cdef5954/service/src/main/kotlin/fi/espoo/evaka/messaging/message/Message.kt#L17